.hyperloop/checks/check-branch-rebases-cleanly.sh

```
@@ -0,0 +1,121 @@
+  # Abort any in-progress rebase inside the worktree before removing it.
+  git -C "$WORKTREE_PATH" rebase --abort 2>/dev/null || true
+  git worktree remove --force "$WORKTREE_PATH" 2>/dev/null || true
+  git branch -D "$TEMP_BRANCH" 2>/dev/null || true
+  rm -rf "$WORK_DIR"
+}
+trap cleanup EXIT
+
+# Add a temporary worktree based on current HEAD.
+git worktree add "$WORKTREE_PATH" -b "$TEMP_BRANCH" HEAD --quiet 2>/dev/null
+
+REBASE_OUTPUT=$(git -C "$WORKTREE_PATH" rebase alpha 2>&1 || true)
```

Shellcheck (0.11.0)

[warning] REBASE_OUTPUT appears unused. Verify use (or export if used externally).

(SC2034)